sway-lsp/src/error.rs

```diff
@@ -33,9 +35,13 @@ pub enum DocumentError {
     ManifestsLockPathFailed { dir: String },
     #[error("Document is already stored at {:?}", path)]
     DocumentAlreadyStored { path: String },
+    #[error("File wasn't able to be created: {:?}", err)]
+    UnableToCreateFile { err: Error },
```

Comment:
nit: could be good to include the filepath in the error

Reply:
good idea.